packages/manager/modules/ng-layout-helpers/src/onboarding/index.less

```diff
@@ -7,17 +7,14 @@
       }
 
       strong {
-        color: rgb(17, 63, 109);
         font-size: 1.125rem;
       }
 
       .oui-icon {
         background-color: transparent;
-        color: #2859c0;
 
         &.oui-icon-guides {
           font-size: 2.2rem;
-          color: #113f6d;
           padding: 0;
           margin-right: 0.3rem;
         }
```

Comment:
We might need to check if those custom styles are still needed given UI-Kit theme has been uniformized